lua/rainbow-delimiters/health.lua

```diff
@@ -168,7 +190,15 @@ function M.check()
 	if queries then
 		start 'Custom queries'
 		for lang, query in pairs(queries) do
-			if lang ~= '' then
+			if lang == '' then
+				if query ~= 'rainbow-delimiters' then
+					local msg = "User defined custom default query.\nIf you meant to use 'rainbow-delimiters', please check your setup."
```

Comment:
I would include the name of the query, then users can check for typos right there instead of having to open their configuration. I also shortened the message a bit. What is your opinion?

```suggestion
					local msg = string.format(
						"User-defined default query '%s'\
						If you meant 'rainbow-delimiters' check for typos",
						query
					)
```

Reply:
This looks good to me. If the default query is setup via a function it will just print a function address, but that is probably rare, and it will still convey the correct idea to the user.